packages/styles/src/utils.ts

```diff
@@ -4,7 +4,7 @@ import { CSSColor } from '@trezor/theme';
 
 export const getValueAndUnit = (valueAndUnit: string): [value: number, unit: string] => {
     const value = parseFloat(valueAndUnit);
-    const [unit] = valueAndUnit.match(/([a-zA-Z]*|%)$/) ?? [''];
+    const [unit] = valueAndUnit.match(/([a-zA-Z]+|%)$/) ?? [''];
```

Comment:
This should behave the same as the previous version because there is the fallback to `''`.